heartbeat/azure-events-az.in

```diff
@@ -761,6 +790,24 @@ def main():
 		longdesc="Set to true to enable verbose logging",
 		content_type="boolean",
 		default="false")
+	agent.add_parameter(
+		"retryCount",
```

Comment:
Parameters should use all lowercase with dashes for multiple words instead of camelcase (to match the other agents), and it would be nice to use parameter names like retries and sleep like we do in the AWS agents:
https://github.com/ClusterLabs/resource-agents/pull/1936/files

Dont change eventTypes though, as that will cause issues for users upgrading.